drftest/base_view_test.py

```diff
@@ -197,11 +200,24 @@ def _generate_docs(self, response, method, data, url_kwargs, format, headers, us
                 'app_name': app_name
             },
             'response': {
-                'data': self._ensure_json_serializable(response_data),
+                'data': response_data,
+                'content_type': response['content-type'],
                 'status': response.status_code,
             }
         })
 
+    def _get_response_data(self, response):
+        if response.get('content-type') == 'application/json':
+            data = response.data if hasattr(response, 'data') else {}
+            return self._ensure_json_serializable(data)
+        if response.get('content-type') == self.XLSX_RESPONSE_CONTENT_TYPE:
+            return self._generate_object_from_byte(response.content)
+
+    def _generate_object_from_byte(self, bytes):
```

Comment:
Nice. I think `_generate_excel_from_bytes` might be better, isn't it?

Reply:
Done [here](7029dd1)